contracts/extbridge/ext_bridge.sol

```diff
@@ -1,12 +1,13 @@
 pragma solidity ^0.4.24;
 
-import "../bridge/Bridge.sol";
+import "../bridge/BridgeTransferERC20.sol";
+import "../bridge/BridgeTransferERC721.sol";
 import "./callback.sol";
 
-contract ExtBridge is Bridge {
+contract ExtBridge is BridgeTransferERC20, BridgeTransferERC721 {
```

Comment:
How about adding a comment that this solidify file is an example to extend bridge contract?

Reply:
How about below commnet? :)
```
// ExtBridge is an extended bridge contract example inherited by BridgeTransferERC20 and BridgeTransferERC721.
// This contract overrides handleERC20Transfer and handleERC721Transfer to make internal call to callback contract.

```

Reply:
Good!